src/logger.js

```diff
@@ -6,8 +6,8 @@ import { request, isBrowser, promiseDebounce, noop, safeInterval, objFilter } fr
 import { DEFAULT_LOG_LEVEL, LOG_LEVEL_PRIORITY, AUTO_FLUSH_LEVEL, FLUSH_INTERVAL } from './config';
 import { LOG_LEVEL, PROTOCOL } from './constants';
 
-type Payload = { [string] : string };
-type Transport = ({ url : string, method : string, headers : Payload, json : Object }) => ZalgoPromise<void>;
+type Payload = { [string] : string | boolean };
+type Transport = ({| url : string, method : string, headers : { [string] : string }, json : Object |}) => ZalgoPromise<void>;
```

Comment:
for headers, allowing it to be part of Payload type would have affected belter down the line. So instead, i just used the `{ [string] : string }` type